alvr/server_openvr/cpp/alvr_server/Controller.cpp

```diff
@@ -284,18 +285,39 @@ bool Controller::onPoseUpdate(float predictionS, FfiHandData handData) {
         pose.vecPosition[1] = p[1];
         pose.vecPosition[2] = p[2];
 
-        pose.vecVelocity[0] = 0;
-        pose.vecVelocity[1] = 0;
-        pose.vecVelocity[2] = 0;
+        // If possible, use the last stored m_pose and timestamp
+        // to calculate the velocities of the current pose.
+        double calcLinearVelocity[3] = { 0.0, 0.0, 0.0 };
```

Comment:
"calc" is a verb/action but you want to describe a variable/data. You can just call it "linearVelocity" because there are no conflicts in this scope. Same for "angularVelocity".